catalog/cataloger_diff.go

```diff
@@ -439,6 +473,12 @@ func createDiffResultsTable(ctx context.Context, executor sq.Execer) (string, er
 		diff_type integer NOT NULL,
 		path character varying COLLATE "C" NOT NULL,
 		entry_ctid tid
+		physical_address character varying,
```

Comment:
What is the cost of adding 6 potentially NULL fields?

Reply:
from what I understood in this case 1 byte per record - having bit mask per record to the fields used